e2e/test/scenarios/admin-2/whitelabel.cy.spec.js

```diff
@@ -225,6 +227,10 @@ describeEE("formatting > whitelabel", () => {
         undoToast().icon("close").click();
 
         cy.log("test uploading a valid image file");
+        cy.findByTestId("login-page-illustration-setting")
+          .findByRole("searchbox", { name: "Login page" })
+          .click();
+        mantinePopover().findByText("Custom").click();
```

Comment:
As previously mentioned, we need to select `Custom` to be able to click the hidden file input.